tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -52,11 +52,11 @@ Feature: restrict Sharing
     And user "Carol" accepts the share "Shares/simple-folder" offered by user "Brian" using the sharing API in the server
     Then as "Carol" folder "/Shares/simple-folder" should exist in the server
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Forbid sharing with groups
+#   allow-shares-with-groups-members configuration is not available now in the oCIS webUI
```

Comment:
I didn't mention the issue link in comment because at the top @issue-ocis-1317 tag is present which is related to the issue